### PR TITLE
chore: add lit components

### PR DESCRIPTION
**Pull request by anubra266**
Add lit components

### CODE REVIEW

File: packages/machines/carousel/src/carousel.types.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export type State = S.State<MachineContext, MachineState>
 
 export type Send = S.Send<S.AnyEventObject>
 
-export type Service = Machine<MachineContext, MachineState, S.AnyEventObject>
+export type Service = Machine<MachineContext, MachineState>
```
Comment (anubra266) [inline packages/machines/carousel/src/carousel.types.ts:100]:
```suggestion
export type Service = Machine<MachineContext, MachineState, S.AnyEventObject>
```

File: packages/machines/avatar/src/avatar.types.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ export type State = S.State<MachineContext, MachineState>
 
 export type Send = S.Send<S.AnyEventObject>
 
-export type Service = Machine<MachineContext, MachineState, S.AnyEventObject>
+export type Service = Machine<MachineContext, MachineState>
```
Comment (anubra266) [inline packages/machines/avatar/src/avatar.types.ts:51]:
```suggestion
export type Service = Machine<MachineContext, MachineState, S.AnyEventObject>
```

File: packages/machines/accordion/src/accordion.types.ts [hunk 3]
```diff
@@ -90,7 +90,7 @@ export type State = S.State<MachineContext, MachineState>
 
 export type Send = S.Send<S.AnyEventObject>
 
-export type Service = Machine<MachineContext, MachineState, S.AnyEventObject>
+export type Service = Machine<MachineContext, MachineState>
```
Comment (anubra266) [inline packages/machines/accordion/src/accordion.types.ts:93]:
```suggestion
export type Service = Machine<MachineContext, MachineState, S.AnyEventObject>
```

File: examples/lit-ts/src/normalize-props.ts [hunk 4]
```diff
@@ -43,6 +43,11 @@ export const normalizeProps = createNormalizer<any>((props: Dict) => {
       continue
     }
 
+    if (key === "disabled") {
+      normalized["disabled"] = value ? "" : undefined
+      continue
+    }
+
```
Comment (segunadebayo) [inline examples/lit-ts/src/normalize-props.ts:50]: You can remove this now. I made the function a bit more robust.

Besides that, the PR is ready to be merged.

**Pull request merged**